invoicer_project/static/js/registration_request.js

```diff
@@ -225,7 +225,34 @@ function validate_name_and_surname() {
 }
 
 
-
+const login = async (url, dataToSend) => {
+    let flag = false, statusCode;
+    try {
+         const res = await fetch(url, {
+             method: 'POST',
+             body: dataToSend
+         })
+        const jsonResponce = await res.json(); //tokens here (if status ok), else error message
+        statusCode = res.status;
+        
+        if (statusCode === 200) {
+            window.localStorage.setItem('accessToken', jsonResponce['access']);
+            window.localStorage.setItem('refreshToken', jsonResponce['refresh']);
+            return true;
+        } else if (statusCode === 400) {
+            emailForm.setAttribute("error", "true");
+            passwordForm.setAttribute("error", "true");
```

Comment:
please move set error attribute for emailForm and passwordForm outside else if

Reply:
didn't get the idea
Should it be like this?
![image](https://user-images.githubusercontent.com/99320958/218065213-d62fbaf7-e219-451a-8085-07961e799259.png)